adaptors/modbus/deploy/e2e/dl_modbus_simulate_thermometer_rtu.yaml

```diff
@@ -26,41 +23,75 @@ spec:
           stopBits: 2
           dataBits: 8
           baudRate: 19200
+          syncInterval: 15s
       properties:
+        - name: switch
+          description: switch to mock or not.
+          type: boolean
+          accessModes:
+            - WriteMany
+            - ReadOnce
+          visitor:
+            register: CoilRegister
+            offset: 1
+            quantity: 1
         - name: temperature
           description: temperature value, the source is in kevin degree.
-          readOnly: true
+          type: float
+          accessModes:
+            - ReadMany
           visitor:
             register: HoldingRegister
             offset: 0
             quantity: 2
-            orderOfOperations:
+            arithmeticOperations:
```

Comment:
`orderOfOperations` is more preferred since the following arithmetic operation is done in sequential order.

Reply:
I change the `orderOfOperations` to `arithmeticOperations` for the following reasons:

1. Array type is in order, so I don't think `orderOf` should be used as a prefix, just like the `selector`, we don't need an `and` prefix.
2. Because there are non-numeric properties, `arithmetic` is used here to indicate that the operation is for a numeric type. In the future, we can use `stringOperations` to indicate the operations of string property and `byteOperations` to indicate the operations of the byte array property.